omod/src/main/resources/apps/dhsregistrationapp_registerPatient.json

```diff
@@ -0,0 +1,127 @@
+                            }
+                        ]
+                    },
+                     {
+                            "legend": "registrationapp.patient.phone.label",
+                            "id": "phoneNumberLabel",
+                            "fields": [
+                                {
+                                    "type": "personAttribute",
+                                    "label": "registrationapp.patient.phone.question",
+                                    "formFieldName": "phoneNumber",
+                                    "uuid": "14d4f066-15f5-102d-96e4-000c29c2a5d7",
+                                    "widget": {
+                                        "providerName": "uicommons",
+                                        "fragmentId": "field/text",
+                                        "config": {
+                                            "regex": "^[0-9]{10}$|^$"
+                                        }
+                                    },
+                                    "cssClasses": ["phone"]
+                               
+                                }
+                            ]
+                        },
+                        {
+                            "id" : "EmailAddress",
+                            "legend" : "Email Address",
+                            "fields" : [
+                                {
+                                    "type" : "personAttribute",
+                                    "label" : "Email Address",
+                                    "formFieldName" : "Email Address",
+                                    "uuid" : "7d35a2d4-26a8-4917-86dc-5ac7b81ac541",
+                                    "widget" : {
+                                        "providerName" : "uicommons",
+                                        "fragmentId" : "field/text",
+                                           "config": {
+                                           "regex": "^[0-9]{10}$|^$"
+                                        }
+                                    }
+                                }
+                            ]
+                        },
+       
+                         {
+                            "id" : "ID CARD",
+                            "legend" : "ID Number",
+                            "fields" : [
+                                {
+                                    "type" : "personAttribute",
+                                    "label" : "Identifier",
+                                    "formFieldName" : "ID Number",
+                                    "uuid" : "7d35a2d4-26a8-4917-86dc-5ac7b81ac541",
+                                    "widget" : {
+                                        "providerName" : "uicommons",
+                                        "fragmentId" : "field/text",
+                                    "config": {
+                                   "regex": "^[0-9]{10}$|^$"
```

Comment:
ID Card will be any of the national ID that is configurable and the value is not limited to numbers only.
This is a quick review comment. Need to understand the module and changes more. 

One Drop down to choose a ID type.
One Text field for the value which is free text.

Reply:
This is something also i wanted to get . True we need to have drop down for id cards, can you specify examples of Id types to include in the dropdown menus

Reply:
ID TYPES : Aadhaar Card, Passport, Driving License, Pan Card, Others.
@sherrif10 ,
I have listed few ID types that will be used. It will be better if we can have it as configurable in admin screen.

Reply:
Thanks @sharvanan, i will implement that in the id card field

Reply:
cc @prapakaransp , @sharvanan , do you agree with me merging this, this has no effect on the server, we only did this to have fully config file , however we  need  to add files to adopt the dropdown, i hope this need t be merged. Let me know your suggestions please thank, Am sorry for lost due to internet shutdown in uganda, i thank God am back